integration-tests/__tests__/__snapshots__/expect-async-matcher.test.js.snap

```diff
@@ -30,23 +30,49 @@ Object {
 
   ● fail with expected promise values
 
-    Error
-      Error: Expected value to have length:
-        2
-      Received:
-        1
-      received.length:
-        1
+    Expected value to have length:
+      2
+    Received:
+      1
+    received.length:
+      1
+
+      22 | 
+      23 | it('fail with expected promise values', async () => {
+    > 24 |   await (expect(Promise.resolve([1])): any).resolves.toHaveLengthAsync(
+         |                                                      ^
+      25 |     Promise.resolve(2)
+      26 |   );
+      27 | });
+      
+      at packages/expect/build/index.js:165:22
+      at __tests__/failure.test.js:24:54
+      at __tests__/failure.test.js:11:191
+      at __tests__/failure.test.js:11:437
+      at __tests__/failure.test.js:11:99
```

Comment:
This is so long since the matcher is defined in that file - if the matcher came from e.g. `node_modules` it'd be shorter